samples/sampler/stories/qa/ContextualPopupDecorator.js

```diff
@@ -258,4 +268,18 @@ storiesOf('ContextualPopupDecorator', module)
 		() => (
 			<ContextualPopupWithArrowFunction />
 		)
+	)
+	.add(
+		'with increment slider',
+		() => (
+			<div style={{textAlign: 'center', marginTop: ri.unit(180, 'rem')}}>
+				<ContextualPopupWithActivator
+					direction="down"
+					popupComponent={renderSliderPopup}
+					spotlightRestrict="self-only"
```

Comment:
```suggestion
					direction={select('direction', ['up', 'down', 'left', 'right'], Config, 'down')}
					popupComponent={renderSliderPopup}
```
If this value is not required, how about making it optional?

Reply:
It's fixed.